auction-server/src/auction.rs

```diff
@@ -1097,10 +1105,26 @@ pub async fn handle_bid_svm(
         .as_ref();
 
 
-    let (bid_amount, permission_key) =
+    let (bid_amount, permission_key, deadline) =
         extract_bid_data_svm(chain_store, bid.transaction.clone(), &chain_store.client).await?;
 
     let bytes_permission_key = Bytes::from(&permission_key.0);
+    let minimum_bid_life_time = match chain_store
+        .get_submission_state(store_new.clone(), &bytes_permission_key)
+        .await
+    {
+        SubmitType::SubmitByServer => BID_MINIMUM_LIFE_TIME_SVM_SERVER,
+        SubmitType::SubmitByOther => BID_MINIMUM_LIFE_TIME_SVM_OTHER,
+        SubmitType::Invalid => BID_MINIMUM_LIFE_TIME_SVM_INVALID,
```

Comment:
this logic is only to handle SubmitByServer and SubmitByOther lifetimes. We should not check this lifetime if the SubmitType is invalid. Right now, in this branch, we return an error which is not very helpful and should be handled by `verify_signatures_svm`

Reply:
this actually shouldn't return an error here, it leaves the error for `verify_signatrues_svm`, because `BID_MINIMUM_LIFE_TIME_SVM_INVALID` is 0. if you have suggestions on better ways to handle this, we can do that too, i agree that this feels a bit weird right now